src/datastore/postgres/schema/v8.ts

```diff
@@ -0,0 +1,59 @@
+import { PoolClient } from "pg";
+import { Logging } from "matrix-appservice-bridge";
+
+const log = Logging.get('postgres/schema/v8');
+
+function domainSetToValues(domains: string[], count: number): [string, Array<string|number|null>] {
+    let res = "";
+    const values: Array<string|number|null> = [];
+    // VALUES ...
+    for (let index = 0; index < domains.length; index++) {
+        const i = index * 3;
+        if (res) {
+            res += ", ";
+        }
+        res += `($${i}, $${i+1}, $${i+2})`;
```

Comment:
The value placeholders of the `pg` library are one-based, not zero-based.
```suggestion
        res += `($${i+1}, $${i+2}, $${i+3})`;
```

Reply:
ah, of course